lib/Convos/Core/Conversation.pm

```diff
@@ -35,7 +38,7 @@ sub _calculate_unread_p {
 
 sub TO_JSON {
   my ($self, $persist) = @_;
-  my %json = map { ($_, $self->$_) } qw(frozen name topic unread);
+  my %json = map { ($_, $self->$_) } qw(frozen name topic unread notifications);
```

Comment:
I forgot to mention something else: I would appreciate if "things" where sorted. This goes for methods and attributes as well.

Also, if you could run the code through `perltidy`, then it would be less work afterwards, and you would get mentioned with "git blame".

I'm going to take this change either way, but I would greatly appreciate it.

Reply:
Righto, I shall try and remember that (updated)